### PR TITLE
Added Package.swift file

### PR DESCRIPTION
**Pull request by ale-gen**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,70 @@
+// swift-tools-version: 5.9
```
Comment (adil-hussain-84) [inline Package.swift:1]: Based on the description of this property on line 2 (i.e. it's "the minimum version of Swift required to build this package"), do we need such a high Swift tools version?

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    ],
+    dependencies: [],
+    targets: [
+        .binaryTarget(
+            name: "JRE_Core",
+            url: "https://sdk-artifactory.justride.com/artifactory/ios-artifacts/JRE_Core/2023.09.26/JRE_Core.xcframework.zip",
+            checksum: "d60f11060d306365af97e98ba8f785259222698f752b5d6dded8f18fc5fabd21"
+        ),
+        .binaryTarget(
+            name: "JSR305",
```
Comment (adil-hussain-84) [inline Package.swift:45]: You have no dependency relationships defined between these products/targets. I think that's fine though for how we're going to link to these libraries from the Justride SDK. If the absence of the dependency relationships causes a problem down the line, we can deal with it then.

File: README.md [hunk 3]
```diff
@@ -1 +1,3 @@
-# j2objc-dynamic-frameworks-swift-package
\ No newline at end of file
+**Swift Package repository for Masabi's dynamic frameworks of J2ObjC libraries**
```
Comment (adil-hussain-84) [inline README.md:1]: Change this to a heading. That is, use the `#` markdown character.

**Review by adil-hussain-84 — APPROVED** [review]

File: Package.swift (hunk 1 above)
Comment (ale-gen) [reply Package.swift:1]: This was the default one for the newest Xcode, we can go down, but how much do we want?

File: Package.swift (hunk 2 above)
Comment (ale-gen) [reply Package.swift:45]: Yes, for our purpose it should be okay, but it's definitely sth to consider when problems occur

**Pull request merged**